copy.go

```diff
@@ -88,6 +88,9 @@ func GatherCopyMetrics(table string, db *sql.DB, start chan time.Time, done chan
 				if copy.Status == "LOADED" {
 					successGauge.WithLabelValues(copy.Table, copy.Schema, copy.Database, copy.Status).Set(1)
 				}
+				if (copy.Status == "LOAD_FAILED" || copy.Status == "PARTIALLY_LOADED" || copy.Status == "LOAD_SKIPPED"){
```

Comment:
I would probably just make this else not else if

Reply:
The reason I did another if is because there is a status for LOAD_IN_PROGRESS, and I was worried that that could trigger the alert when there's nothing wrong and something is just in the process of loading @chriskuchin . Thoughts on that?

Reply:
I think you handle that in your alert with a for... the bad status needs to persist for some amount of time before triggering an alert

Reply:
If you are really worried i would do an else if ! LOAD_IN_PROGRESS but that's technically a lie since the job hasn't succeeded yet

Reply:
Also the metrics have a tag for status so you can filter out the load_in_progress status anyways

Reply:
I think you are right, the range in the for loop would handle it. I will change it to an else